test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
```

Comment:
Why is this necessary?

Reply:
I was testing and my test env was loading TT1 by default, which is not a block theme, and it was making the test fail. I've seen other tests do this same thing and since we rely on having a block theme installed, it felt like the correct way to go about it? I'm not sure!

Reply:
Thanks. That's useful context.

Reply:
We should probably explain that in a comment.

Reply:
Yeah, due to WordPress being a stateful environment, we kind of have to reset the state after each/all tests.